transaction-view/src/transaction_meta.rs

```diff
@@ -424,4 +453,43 @@ mod tests {
             assert!(iter.next().is_none());
         }
     }
+
+    #[test]
+    fn test_address_table_lookup_iter_empty() {
+        let tx = simple_transfer();
+        let bytes = bincode::serialize(&tx).unwrap();
+        let meta = TransactionMeta::try_new(&bytes).unwrap();
+
+        // SAFETY: `bytes` is the same slice used to create `meta`.
+        unsafe {
+            let mut iter = meta.address_table_lookup_iter(&bytes);
+            assert!(iter.next().is_none());
+        }
+    }
+
+    #[test]
+    fn test_address_table_lookup_iter() {
```

Comment:
You're missing a test for >1 ATL lookups

Reply:
also for instructions!